indexing-service/src/main/java/io/druid/indexing/common/task/RealtimeIndexTask.java

```diff
@@ -107,12 +112,24 @@ private static String makeDatasource(FireDepartment fireDepartment)
   @JsonIgnore
   private volatile QueryRunnerFactoryConglomerate queryRunnerFactoryConglomerate = null;
 
+  @JsonIgnore
+  private volatile IndexMerger indexMerger;
```

Comment:
actually nevermind, it looks like if someone were to change the value it would get used as expected